### PR TITLE
manually run extension api gen for release

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

**Comment by calebmshafer** [comment]
@aruniverse can you target `master` first?

**Review by kckst8 — APPROVED** [review]

**Comment by aruniverse** [comment]
@mergifyio backport master

File: core/extension/index.d.ts [hunk 1]
```diff
@@ -49,11 +49,12 @@ export {
 	MessageBoxType,
 	MessageBoxIconType,
 	MessageBoxValue,
+	SelectionSetEventType,
```
Comment (pmconne) [inline core/extension/index.d.ts:52]: is the sorting arbitrary? makes difficult to tell what actually changed vs what moved.

Comment (aruniverse) [reply core/extension/index.d.ts:52]: agreed, i was initially thinking my local changes were just re-ordering but there were a few new additions to the api. looking into seeing if we can keep it sorted & stable...

Comment (calebmshafer) [reply core/extension/index.d.ts:52]: @johnnyd710 @aruniverse can you guys file a bug to fix the ordering problem?

@aruniverse let's just manually fix things for this release.

**Review by pmconne — APPROVED** [review]

**Comment by pmconne** [comment]
@calebmshafer @skirby1996 do you have any idea why @mergifyio always re-requests review from itwinjs-core-backport-reviewers immediately after I approve one of its PRs?

**Comment by calebmshafer** [comment]
> @calebmshafer @skirby1996 do you have any idea why @Mergifyio always re-requests review from itwinjs-core-backport-reviewers immediately after I approve one of its PRs?

@pmconne not sure why it's not picking up the fact it already reviewed and requested it. @skirby1996 can look into it once he's back on thursday.

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**